IDEAS/Buildings/Components/Interfaces/RectangularZoneTemplateInterface.mo

```diff
@@ -1021,6 +1024,10 @@ initial equation
   if hasIntFlo then
     assert(nExtFlo == 1, "The parameter nExtFlo should be = 1 when internalWall type is chosen for Floor");
   end if;
+
+  if hasWinCei then
+    assert(abs(ACei-ACeiNet) < 0.001, "The overwrite parameter ACei should not be used when the roof has a window");
```

Comment:
why not? 

also, this assert looks like it will always fail when `hasWinCei=true`?

Reply:
I think this should be `assert(abs(ACei-w*l) < 0.001, "The overwrite parameter ACei should not be used when the roof has a window");`?